.github/workflows/openms_ci_matrix_full.yml

```
@@ -245,6 +245,27 @@ jobs:
+      if: startsWith(matrix.os, 'macos')
+      uses: conda-incubator/setup-miniconda@v3
+      with:
+        auto-update-conda: true
+        python-version: 3.9
+        activate-environment: openms-parquet
+        use-mamba: true
+
+    - name: Install Parquet dependencies (macOS only)
+      if: startsWith(matrix.os, 'macos')
+      shell: bash -el {0}
+      run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:5:67: Double quote to prevent globbing and word splitting

(shellcheck)